docs/docs/api/index.md

```diff
@@ -0,0 +1,82 @@
+# Mathesar's API
+
+Mathesar has an API available at `/api/rpc/v0/` which follows the [JSON-RPC](https://www.jsonrpc.org/specification) spec version 2.0. It has been built primarily for consumption by the Mathesar front end but theoretically could be used to build automation workflows and Mathesar integrations.
+
+## Caveats
+
+- The Mathesar API is **not yet stable**. If you build logic that depends on it, be mindful that future Mathesar versions will likely bring breaking changes to the API without warning or notice.
+
+- This API _documentation_ is nascent and somewhat rough. We encourage you to report any docs inaccuracies via GitHub [issues](https://github.com/mathesar-foundation/mathesar/issues).
```

Comment:
Nit: this phrasing is a bit idiomatic. I'll offer an alternative:

> This API documentation is still in its early stages and may contain inaccuracies or incomplete information. If you encounter any issues, please report them via GitHub.

Reply:
Thank you. In our pairing session, Brent and I had changed this to:

> - This API _documentation_ is somewhat rough. We encourage you to report any docs inaccuracies via GitHub [issues](https://github.com/mathesar-foundation/mathesar/issues).

But yours is even better!

Fixed in e7c0df99109a32cdbababa5085bebe2641b16c64